pydundas/dundas.py

```diff
@@ -2,9 +2,36 @@
 import logging
 from .exceptions import YamlNotReadableError, LoginFailedError
 
+
+# Helper method, not really part of the class itself.
+def creds_from_yaml(yamlpath):
```

Comment:
Bit late, but this might have been a nice candidate for a [NamedTuple](https://docs.python.org/2/library/collections.html#collections.namedtuple). Although `credentials` is probably not an appropriate name when the url is included :disappointed: